msal/src/main/java/com/microsoft/identity/client/Account.java

```diff
@@ -97,4 +98,20 @@ String getEnvironment() {
     public Map<String, ?> getClaims() {
         return mIdTokenClaims;
     }
+
+    @NonNull
+    @Override
+    public String getUsername() {
```

Comment:
nit: should we add a comment stating that as of now, this field can be empty in B2C scenario?

Reply:
Sure I'll add a comment

Reply:
Resolved in 07583c6db89fb6712643dd07845c05f4143777f3